src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,19 @@
+import { NavLink } from 'react-router-dom';
+import './navBarStyle.css';
+
+const NavBar = () => {
```

Comment:
what do you think if we just use 

```
const NavBar = () => (
<nav className="nav-container">
      <NavLink className="nav-link" exact to="/list">
          List
      </NavLink>
      <NavLink className="nav-link" exact to="/addItem">
           Add Item
      </NavLink>
    </nav>
  )

```

Reply:
the reason is that currently we aren't used any logic yet inside of this component